docs/commands/look-detection.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Look Detection
+category: Techniques
+mention:
+  - BedrockCommands
+  - AjaxGb
+  - Plagiatus
+  - zheaEvyline
+description: This command-technique allows you to detect when a target looks at a player/entity/coordinate and subsequently run your desired commands.
+---
+
+## Introduction
+
+[Sourced By Bedrock Commands Community Discord](https://discord.gg/SYstTYx5G5)
+
+**Credits:** *@AjaxGb*
+
+This command-technique allows you to detect when a target looks at a player/entity/coordinate and subsequently run your desired commands.
+
+## Command
+
+<CodeHeader>mcfunction</CodeHeader>
```

Comment:
I'd recommend adding full path to function file: BP/functions/look_detector.mcfunction

Reply:
And in the following ones

Reply:
It's more of a single command technique that can be applied to many different cases as compared to a system, so I thought it wouldn't be too relevant. Are you sure it's better to do that? 👀